site/routes/definitions.js

```diff
@@ -1,14 +1,27 @@
 // @flow
 
-type RouteDefinition = {|
+type RouteDefinition = {
   title: string | (props: Object) => string,
   key: string,
   route: string,
-  defaults?: any,
   stateToProps?: (state: Object, params?: Object) => any,
   gen?: (state: Object) => Array<Object>,
-  render?: (state: Object) => any,
-|}
+}
+
+const getBadgersByCategory = (badgers, category) => (
+  badgers.filter(badger => badger.categories.filter(cat => cat === category).length > 0)
+);
+
+const createPages = (category, count) => (
+  Array.from(Array(Math.ceil(count / 20)).keys()).map(page => ({ category, page: page + 1 }))
+);
+
+const genBadgersParams = state => (
+  state.categories.reduce((params, category) => {
+    const count = getBadgersByCategory(state.badgers, category).length;
+    return params.concat(createPages(category.toLowerCase(), count));
+  }, createPages('everyone', state.badgers.length + 1))
+);
```

Comment:
@JoeStanton Turning the list of badgers into different pages per category is a UI thing. So it fits better in the `routes` folder than the `state` folder.

I raised a similar issue on the [Events and Jobs dictionary currently stored in State](https://github.com/redbadger/website-honestly/issues/298). The State should just hold arrays and then logic to manipulate the State for the components should sit in the routes folder?

Doesn’t seem right to store the same list in two different ways in State. (I guess I see `state.json` as a denormalised database)